src/management/users.ts

```diff
@@ -1,9 +1,10 @@
 import Client from '../networking';
-import { apply } from '../utils/whitelist';
-import { toCamelCase } from '../utils/camel';
+import {toCamelCase} from '../utils/camel';
 import Auth0Error from './error';
+import {Telemetry} from '../networking/telemetry';
+import {GetUserOptions, PatchUserOptions} from '../types';
 
-function responseHandler(response, exceptions = {}) {
+function responseHandler(response: any, exceptions = {}) {
```

Comment:
We should be able to type response here, but will need to have an update on the networking client as well as the fetchWithTimeout file. It looks like we use fetch, so we should have types for free.